src/main/java/org/betonquest/betonquest/database/MySQL.java

```diff
@@ -198,4 +204,13 @@ private void migration2(final Connection connection) throws SQLException {
             statement.executeUpdate("ALTER TABLE " + prefix + "global_tags " + "ADD PRIMARY KEY (tag)");
         }
     }
+
+    private void migration3(final Connection connection) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            statement.executeUpdate("UPDATE " + prefix + "player_profile " + "SET name = '" + profileInitialName
+                    + "' WHERE name = NULL");
+            statement.executeUpdate("ALTER TABLE " + prefix + "player_profile " + "MODIFY COLUMN name "
+                    + "VARCHAR(63) NOT NULL");
```

Comment:
This will fail in the case that there already exists a profile with the default name. But it shouldn't have happened yet as we didn't add any functionality to manage profiles as far as I know, so this is more of a note than a request for change.

Reply:
I think that we do nothing about this, as users should not have used this until now